vendor/golang.org/x/sys/unix/ztypes_linux_ppc.go

```diff
@@ -417,7 +417,7 @@ const (
 
 type SockaddrStorage struct {
 	Family uint16
-	_      [122]uint8
+	Data   [122]byte
 	_      uint32
 }
 
```

Comment:
The code patch appears to be a simple change to the SockaddrStorage struct definition. Instead of having an unnamed array of 122 uint8 values, it now has a named field called "Data" that is an array of 122 bytes. This change shouldn't introduce any bugs, but it might improve readability of the code in some contexts by allowing access to the underlying byte data without needing to use unsafe pointer casts. Overall, this change seems safe and reasonable.